<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createSim</class>
 <widget class="QDialog" name="createSim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>490</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Etat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>10</y>
     <width>118</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Voisinage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>122</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transition</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="valeur8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="color8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Valeur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Couleur</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>150</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="vonNeumann">
      <property name="text">
       <string>Von Neumann</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="vonNeumann2">
      <property name="text">
       <string>Von Neumann généralisé</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="moore">
      <property name="text">
       <string>Moore</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="moore2">
      <property name="text">
       <string>Moore généralisé</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="arbitraire">
      <property name="text">
       <string>Arbitraire</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="rayonBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rayon du voisinage</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>104</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="golChecked">
      <property name="text">
       <string>Game Of Life</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="langstonChecked">
      <property name="text">
       <string>Langston's Loop</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="brainChecked">
      <property name="text">
       <string>Brian's Brain</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="griffithChecked">
      <property name="text">
       <string>Griffeath</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="manuelChecked">
      <property name="text">
       <string>Manuel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>34</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Etat 1 </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Etat 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Etat 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Etat 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Etat 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Etat 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Etat 7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Etat 8 </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>87</width>
     <height>260</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QRadioButton" name="gol3">
    <property name="text">
     <string>Game Of Life</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="brain3">
    <property name="text">
     <string>Brian's Brain</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="griffeath3">
    <property name="text">
     <string>Griffeath</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>createSim</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>createSim</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
